slash/react/src/Form/ChildrenQuestion/ChildrenQuestion.tsx

```diff
@@ -0,0 +1,12 @@
+import "@axa-fr/design-system-slash-css/dist/Form/ChildrenQuestion/ChildrenQuestion.scss";
```

Comment:
Add unit test for the component

Reply:
![image](https://github.com/user-attachments/assets/586369cc-fcdd-48fe-a6c4-a5ffa3b6bd30)